vwifi.c

```diff
@@ -506,6 +548,54 @@ static void inform_bss(struct vwifi_vif *vif)
     }
 }
 
+/* Helper function that prepares a structure with self-defined BSS information
+ * and "informs" the kernel about the "new" Independent BSS.
+ */
+static void ah_inform_bss(struct vwifi_vif *vif)
```

Comment:
Nearly all of the cfg80211 kernel functions in adhoc mode are named with *ibss*. We should follow the naming scheme.